src/f32/quat.rs

```diff
@@ -358,6 +358,64 @@ impl Quat {
         }
     }
 
+    /// Performs a spherical linear interpolation between `self` and `other`
+    /// based on the value `s`.
+    ///
+    /// When `s` is `0.0`, the result will be equal to `self`.  When `s`
+    /// is `1.0`, the result will be equal to `other`.
+    #[inline]
+    pub fn slerp(self, end: Self, s: f32) -> Self {
+        // http://number-none.com/product/Understanding%20Slerp,%20Then%20Not%20Using%20It/
```

Comment:
I was going to add slerp when I wrote glam, read this article and decided not to :) Well, partially that was the reason. I was going for a minimal feature set which was somewhat based on what https://github.com/nfrechette/rtm supported. It also doesn't have a slerp. Are you using slerp for something?

Reply:
Yes. I was looking at migrating some stuff from cgmath to glam and slerp was missing. I was checking animations with some of the [glTF sample models](https://github.com/KhronosGroup/glTF-Sample-Models) and the lack of constant velocity makes `lerp` unsuitable.

Reply:
Thanks, that's good to know.